src/main/java/org/example/PerformWordSearch.java

```diff
@@ -23,22 +22,31 @@ public PerformWordSearch(String inputFilePath, String searchWord) {
     public void run() {
         int matchingResultsCount = 0;
         String inputDataFromFile;
+        String matchingResult;
+        String wordMessage;
         try {
             inputDataFromFile = new String(Files.readAllBytes(Paths.get(inputFilePath)));
             inputDataFromFile = inputDataFromFile.replaceAll("[^a-zA-Z0-9@-]", "  ");
```

Comment:
Please take the string "[^a-zA-Z0-9@-]" to Constants class and access it from there

Reply:
Updated.